datafusion/core/tests/dataframe/mod.rs

```diff
@@ -1489,7 +1489,7 @@ async fn test_read_batches_empty() -> Result<()> {
     let state = SessionState::new_with_config_rt(config, runtime);
     let ctx = SessionContext::new_with_state(state);
 
-    let schema = Arc::new(Schema::new(vec![
+    let _schema = Arc::new(Schema::new(vec![
```

Comment:
I think you could simply delete this statement entirely as the Schema is not needed

Reply:
done